tests/Common/DevBranchesTest.php

```diff
@@ -40,15 +40,43 @@ public function testCreateBranch(Client $providedClient)
         $this->assertSame(['id', 'name', 'email'], array_keys($branch['admin']));
         $this->assertSame($adminId, $branch['admin']['id']);
 
-        $this->expectException(ClientException::class);
-        $this->expectExceptionMessage(sprintf('There already is a branch with name "%s"', $branchName));
         $event = $this->findLastEvent($providedClient, [
             'event' => 'storage.devBranchCreated',
             'objectId' => $branchId
         ]);
         $this->assertSame($branchName, $event['objectName']);
 
         $branches->createBranch($branchName);
```

Comment:
Já se jen zeptám. Zkoušeli jste si oba dva pouštět ten test že jo? On totiž nefunguje. Tenhle řádek se musí smazat.

Reply:
Jo ešte som si ho nespúšťal, prechádzal som len ten kód

Reply:
To bude moje chyba, já jsem to celé otestoval a pak jsem to ještě rebasnul a pak už jsem to nespustil.

Reply:
Fixed